### ISSUE TITLE
Early stop with low dice coefficient

### ISSUE THREAD

**Issue by muween** [ellisdg/3DUnetCNN#194]
I use the training.py to train the brats2018 dataset precessed by N4. The training process was very slow, and the dice didn't increase anymore when it come to 0.45 util the program early stopped itself. Is there something wrong with my keras version? My keras is 2.2.4 , the backup is tensorflow 1.12.0.  Or it is related to the parameters setting or data process?

**Comment by huawenqing** [comment]
Ooo, it's normal for original 3D unet

**Comment by RafiaRahim** [comment]
you can increase initial learning rate for better learning, i have already tried this. Also you can change early stop value in config settings.

**Comment by huawenqing** [comment]
> you can increase initial learning rate for better learning, i have already tried this. Also you can change early stop value in config settings.

What about your final output of dice?

**Comment by tsaiwentage** [comment]
i got the final dice coef on validation dataset of 0.56, with the original hyperparameter of the original 3d-unet

**Comment by tsaiwentage** [comment]
maybe you change the hyperparameter of the original config, I use the original config of the original 3d-unet to train on rtx 2080ti, the result is similar to the author showed on readme. I got the dice coefficient of 56 on the validation set. And now, i'm training the isensee's version, after 94 epoches, the dice coef is raise to 64

**Comment by sneh-debug** [comment]
> i got the final dice coef on validation dataset of 0.56, with the original hyperparameter of the original 3d-unet

and what about validation score of Whole ,enhancing and core?
